app/assets/locales/messages_ccodk_default.txt

```diff
@@ -289,7 +289,8 @@ notification.sync.connections.action=You don't have any active data connections
 
 #Add new localization strings here
 
-
+callout.failure.dialer=Dialer app not found

```

Comment:
The SMS one I think will make sense to a lot of users, but I think "Dialer app" might be a bit confusing for folks.

Can we make this something that end-users might be more familiar with, like

No calling app found. The current device is not currently configured to make telephone calls.

or something?